mobilecoind/src/config.rs

```diff
@@ -87,8 +90,8 @@ pub struct Config {
 
     /// Fog ingest enclave CSS file (needed in order to enable sending
     /// transactions to fog recipients).
-    #[clap(long, value_parser = load_css_file, env = "MC_FOG_INGEST_ENCLAVE_CSS")]
-    pub fog_ingest_enclave_css: Option<Signature>,
+    #[clap(long, value_parser = load_css_file, env = "MC_FOG_INGEST_ENCLAVE_CSS", use_value_delimiter = true)]
+    pub fog_ingest_enclave_css: Vec<Signature>,
```

Comment:
I believe that using `use_value_delimiter=true` is correct here, and will allow that a list like `foo.css,bar.css` can be the value of `MC_FOG_INGEST_ENCLAVE_CSS` environment variable, and then this will be split by clap, and then the arguments passed one by one to `load_css_file`. I did not actually test this, I probably should.

Reply:
okay, i added some more tests of this to build confidence